tests/test_parameter_interpolation.py

```diff
@@ -102,9 +100,7 @@ def test_nonbonded_advanced(self):
         water_coords = self.get_water_coords(D, sort=False)
         coords = water_coords[:size]
         padding = 0.2
-        diag = np.amax(coords, axis=0) - np.amin(coords, axis=0) + padding
-        box = np.eye(3)
-        np.fill_diagonal(box, diag)
+        box = np.eye(3) * 3
 
```

Comment:
add a test for a non-orthogonal box just to sanity check that the test would fail?